### PR TITLE
hide log level flag

### PR DESCRIPTION
**Pull request by makalaaneesh**
Hiding flag because of known issue - https://github.com/yugabyte/yb-voyager/issues/1786

### CODE REVIEW

**Review by priyanshi-yb — APPROVED** [review]
LGTM

File: yb-voyager/cmd/getDataMigrationReportCommand.go [hunk 1]
```diff
@@ -418,6 +418,7 @@ func init() {
 	registerExportDirFlag(getDataMigrationReportCmd)
 	getDataMigrationReportCmd.PersistentFlags().StringVarP(&config.LogLevel, "log-level", "l", "info",
 		"log level for yb-voyager. Accepted values: (trace, debug, info, warn, error, fatal, panic)")
+	getDataMigrationReportCmd.PersistentFlags().MarkHidden("log-level")
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/getDataMigrationReportCommand.go:421]: just curious, if we have defined it on root why do we need on bulk and get command here.

**Review by sanyamsinghal — APPROVED** [review]

File: yb-voyager/cmd/getDataMigrationReportCommand.go (hunk 1 above)
Comment (makalaaneesh) [reply yb-voyager/cmd/getDataMigrationReportCommand.go:421]: Something I wanted to also discuss! - these commands do not call the `registerCommonGlobalFlags` function. Not ideal; we should pick it up sometime.

Comment (priyanshi-yb) [reply yb-voyager/cmd/getDataMigrationReportCommand.go:421]: I think it was intentional that some global flags are not required for these commands e.g. get datamigration one doesn't need the yes or send-diagnostics etc..

**Pull request merged**